airflow/migrations/versions/0032_3_0_0_rename_execution_date_to_logical_date_and_nullable.py

```diff
@@ -49,10 +49,15 @@ def upgrade():
             "execution_date",
             new_column_name="logical_date",
             existing_type=TIMESTAMP(timezone=True),
-            existing_nullable=False,
+            existing_nullable=True,
```

Comment:
```suggestion
            nullable=True,
```

`existing_nullable` is for specifying the nullable property we _don’t_ want to change. Here we want to change nullability.

Reply:
Thanks, I Implemented this change.